osbenchmark/results_publisher.py

```diff
@@ -130,6 +130,8 @@ def publish(self):
 
         for record in stats.op_metrics:
             task = record["task"]
+            self.logger = logging.getLogger(__name__)
+            self.logger.info("Record: [%s], Task: [%s]", record, task)
```

Comment:
Might be better to have this in a separate commit.

Reply:
Good catch, let me remove this!